### PR TITLE
Restructur and start dedicated pages for user tutorials 

### PR DESCRIPTION
**Pull request by 6543**
This ...
- ... moves the Overview into it's own page and make it more readable.
- ... starts the creation of an dedicated page tor users to get started.
- ... add .editorconfig to streamline text format and style
- ... starts the creation of an dedicated page to explain the terms and architecture

### CODE REVIEW

File: docs/01-overview.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 1
+---
+
+# Overview
+
+Stardust XR _(formerly known as Stardust)_ is an XR [display server](https://itsfoss.com/display-server) designed for Linux-based systems<!-- (possibly most unix-based too, but untested)-->.<br/>
+Unlike traditional display servers that focus on 2D screens, Stardust XR introduces new conventions to utilize unbounded 3D space in virtual or real environments.
+
+## Priorities
```
Comment (6543) [inline docs/01-overview.md:10]: @technobaboo please check if i narrowd it down

File: docs/01-overview.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- **Flexible Object Interaction**: Users can choose objects that best suit their current scenario, with easy setup for default configurations.
+
+## Architecture
+
+TODO: add some diagram of the core components needed and how they interface
+<!-- example: https://woodpecker-ci.org/docs/next/usage/terminology#woodpecker-architecture -->
+
+## Projects & Components
+
+- [**telescope**](https://github.com/StardustXR/telescope): bundle most used components to have a demo environment set up
```
Comment (6543) [inline docs/01-overview.md:32]: that is the part where I'm most lacking ... I would say merge this pull and discust this list in a new pull

File: docs/01-overview.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  - Handling data as Items that can represent files, Wayland surfaces, etc.
+
+- **Personal Device Interaction**: Focuses on individual user interactions rather than collaborative 3D spaces.
+
+- **Flexible Object Interaction**: Users can choose objects that best suit their current scenario, with easy setup for default configurations.
+
+## Architecture
+
+TODO: add some diagram of the core components needed and how they interface
+<!-- example: https://woodpecker-ci.org/docs/next/usage/terminology#woodpecker-architecture -->
```
Comment (6543) [inline docs/01-overview.md:28]: I have no idea but @technobaboo if you provide me with an scetch I'll create an mermaidjs out of it

**Comment by 6543** [comment]
> btw we can probably just remove all the stardust schema docs on the site given there's kdl files in the core repo that do that now

ok we should do that in an seperate pull

I still think the official website should contain the protocol documentation too ... but that could be done with code(docs)-generation and in a seperate subcategory later on ...

File: docs/01-overview.md (hunk 1 above)
Comment (technobaboo) [reply docs/01-overview.md:10]: seems pretty good, yea!!

**Comment by technobaboo** [comment]
also also keep in mind we're probably gonna redo the site entirely soon so any changes we make here don't have to be perfect but are great for if the redo gets delayed or just as a framework

**Comment by 6543** [comment]
sure there still is lot of TODOs in my pull but it at least make things a bit more discoverable

and if you use it to get some ideas for the new docs thats great too :)

**Comment by technobaboo** [comment]
should i just merge this in right now or resolve all the threads?

**Comment by 6543** [comment]
ah I can do that too - I know _some_ git

**Comment by technobaboo** [comment]
i don't mind at all, i'll resolve some threads with more info

File: docs/01-overview.md (hunk 2 above)
Comment (technobaboo) [reply docs/01-overview.md:32]: sure thing

File: docs/01-overview.md (hunk 3 above)
Comment (technobaboo) [reply docs/01-overview.md:28]: image
sorry it took me so long but i got it!

Comment (technobaboo) [reply docs/01-overview.md:28]: it doesn't cover everything but most things it does so yea

File: docs/09-terminology.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Glossary
+
+- **Item**: 
+- **Panel Shells**: 3D UIs around 2D apps
```
[the remark this answers is not in the document]
Comment (technobaboo) [reply docs/09-terminology.md:4]: uhhh yeah for now that's probably the only glossary item that's unknown by XR devs, will need to revisit it though

File: docs/01-overview.md (hunk 3 above)
Comment (6543) [reply docs/01-overview.md:28]: depends on #9

File: docs/09-terminology.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (6543) [reply docs/09-terminology.md:4]: well If you know a good source where the OpenXR glossary is documented I'll just refference it and we can only document stardust specific terms here ... else if such thing does not exist, we can be the list for others ...

_not everybody is an XR dev >D_

**Comment by 6543** [comment]
to #9 I still thin for the time being mrermaidjs support should be added, as it is quite helpfull.

for the diagram you provided, that was easyer to just be provided as an image for now ...

**Comment by 6543** [comment]
@technobaboo I guess this is ready now?

**Pull request merged**